webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -754,6 +755,10 @@ const firstVisibleMsg = {
                 mckInitializeChannel.onTypingStatus(resp);
             },
             onFeedbackClick: function (resp) {},
+            onDisconnect: function () {
+                IS_SOCKET_CONNECTED = false;
```

Comment:
where are these getting used?
and what is the difference between connected and initialized?